assets/tasks/common.json

```diff
@@ -847,6 +847,8 @@
                 "rsync",
                 "-av",
                 "--progress",
+                "-e",
+                "ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null",
```

Comment:
Here better have each word in one line, like for the [others](https://github.com/torizon/vscode-torizon-templates/blob/dev/assets/tasks/common.json#L529)